source/Calamari/Kubernetes/SpecialVariables.cs

```diff
@@ -51,6 +51,14 @@ public static class Helm
             public const string TillerNamespace = "Octopus.Action.Helm.TillerNamespace";
             public const string TillerTimeout = "Octopus.Action.Helm.TillerTimeout";
             public const string ChartDirectory = "Octopus.Action.Helm.ChartDirectory";
+            public const string OciRegistry = "Octopus.Action.Helm.OciRegistry";
+
+            public static class ScriptSource
+            {
+                public const string OciRegistry = nameof(OciRegistry);
```

Comment:
Just a minor thought - consider make this a string literal, so renaming the variable above won't change this value